core/src/main/java/com/scalar/db/api/StorageInfo.java

```diff
@@ -22,6 +22,13 @@ public interface StorageInfo {
    */
   int getMaxAtomicMutationsCount();
 
+  /**
+   * Returns whether the storage guarantees consistent reads for virtual tables.
+   *
+   * @return true if the storage guarantees consistent reads for virtual tables, false otherwise
+   */
+  boolean isConsistentVirtualTableRead();
```

Comment:
```suggestion
  boolean canConsistentVirtualTableRead();
```
or 
`isConsistentVirtualTableReadGuaranteed`?

Reply:
Fixed in 92e5729d112ae559b0a8d7230e74aa78f7cf00e3. Thanks!